pkg/testing/integration/program.go

```diff
@@ -1815,14 +1814,17 @@ func (pt *ProgramTester) copyTestToTemporaryDirectory() (string, string, error)
 		tmpdir = targetDir
 		projdir = targetDir
 	}
+	if wd := pt.opts.RelativeWorkDir; wd != "" {
+		projdir = filepath.Join(projdir, wd)
+	}
 	// Copy the source project.
 	if copyErr := fsutil.CopyFile(tmpdir, sourceDir, nil); copyErr != nil {
 		return "", "", copyErr
 	}
 	// Reload the projinfo before making mutating changes (workspace.LoadProject caches the in-memory Project by path)
```

Comment:
Huh this comment is out of date now, we don't cache by path anymore so this reload probably isn't needed anymore.

Reply:
I'll test that theory :-)

Reply:
The cache is no longer there but projinfo returns a struct with a public field that points to the project directory.
We should probably still reload because it's conceivable that a test case will try to do something inside that project directory.
(Yeah, we could punch just that field, but that's a bit icky since it can't account for any future state that getProjInfo starts tracking.)